spec/006_artists_and_genres_spec.rb

```diff
@@ -12,6 +12,9 @@
         Song.new("The Luckiest Guy on the Lower East Side", artist, genre)
         Song.new("Long-Forgotten Fairytale", artist, other_genre)
 
+        # running into a weird issue where `artist.songs` had nothing in it, and `all` for `Artist` did _not_ get added in and I have no clue why...
+        # my genres method was `@songs.map(&:genre).uniq`
```

Comment:
I'm not sure off hand without taking a deeper look. We can pair on this today if you like!